language/functional_tests/tests/testsuite/discovery/reconfiguration_via_address_rotation.mvir

```diff
@@ -0,0 +1,142 @@
+//! account: alice
```

Comment:
Nit: we should probably call this file `reconfiguration_via_network_address_rotation.mvir` to avoid confusion with account addresses...